src/CattyTests/Parser & Serialization/CatrobatLanguage0.991/XMLParserTests/Unit Tests/XMLParserTests0991.swift

```diff
@@ -558,4 +558,14 @@ class XMLParserTests0991: XMLAbstractTest {
         XCTAssertTrue(disabledBrick.isDisabled)
         XCTAssertFalse(enabledBrick.isDisabled)
     }
+
+    func testSpeakBrickWithoutCommentedOut() {
+        let project = self.getProjectForXML(xmlFile: "SpeakBrickWithoutCommentedOut_0991")
+        let object = project.objectList.object(at: 0) as! SpriteObject
+        let startScript = object.scriptList.object(at: 0) as! Script
+        let speakBrick = startScript.brickList.object(at: 0) as! SpeakBrick
+
+        let xmlElement = speakBrick.xmlElement(for: "SpeakBrick", with: CBXMLSerializerContext())
```

Comment:
This is a parser test, please do not test the serialization. The subsequent two lines can be removed. Please bring back the ones you deleted.

Reply:
@m-herold  are you talking about the line `565 - 566`?
I need spriteBrick to get the `xmlElemnt` of the brick which is then used to test the `commentOut` tag

Reply:
I was talking about the other direction (_#568_ and _#569_). Thanks!

Reply:
It is not necessary to test the serialization (object to xml) which `xmlElement` is doing because this is tested at the serializer tests. For this test it is enough to test that (a) the xml provided by your file can be properly parsed/read, that (b) the brick is correctly parsed (i.e. as SpeakBrick), and (c) ideally that the _commentedOut_ property is false (if not set).